test/test_va_api_fixture.cpp

```diff
@@ -531,6 +531,10 @@ TEST_F(VAAPIFixture, getDisplay)
 
     vaDisplay = getDisplay();
     ASSERT_TRUE(vaDisplay);
+    if (!vaDisplay)
+        return;
```

Comment:
This is unnecessary since the `ASSERT_TRUE(vaDisplay)` will cause the test to abort if `!vaDisplay`